app/views/facility_estimates/_form.html.haml

```diff
@@ -0,0 +1,55 @@
+= simple_form_for @estimate, url: form_url, method: form_method do |f|
+  .inline-form-controls
+    = f.input :name
+    .span-4
+      = f.label :user_id, Estimate.human_attribute_name(:user_id)
+      = f.input_field :user_id,
+        collection: user_collection,
+        class: "js--chosen js--select-with-search",
+        include_blank: false,
+        data: { search_url: search_facility_estimates_path },
+        disabled: f.object.persisted?
+    .span-4
+      = f.input :expires_at, as: :date_picker, min_date: Time.current.iso8601
+  .form-inputs
+    = f.input :note
+
+  #new_estimate_products_table
+    %h3= t("facility_estimates.form.current_products")
+    %table.table
+      %thead
+        %tr
+          %th= EstimateDetail.human_attribute_name(:product)
+          %th= EstimateDetail.human_attribute_name(:quantity)
+          %th= EstimateDetail.human_attribute_name(:duration)
+          %th.text-center= t("shared.remove")
```

Comment:
Is there any reason for adding a header to this column? I find it a bit weird, you already see a Remove button so it's not like it's adding much information